src/components/MarketSelector/types.tsx

```diff
@@ -0,0 +1,34 @@
+import { MarketInfo } from "domain/synthetics/markets";
+import { TokensData } from "domain/synthetics/tokens";
+import { GmTokensFavoritesContextType } from "domain/synthetics/tokens/useGmTokensFavorites";
+
+export type CommonPoolSelectorProps = {
+  label?: string;
+  className?: string;
+  selectedMarketAddress?: string;
+  selectedIndexName?: string;
+  markets: MarketInfo[];
+  marketTokensData?: TokensData;
+  showBalances?: boolean;
+  isSideMenu?: boolean;
+  getMarketState?: (market: MarketInfo) => MarketState | undefined;
+  onSelectMarket: (market: MarketInfo) => void;
+  showAllPools?: boolean;
+  showIndexIcon?: boolean;
+} & GmTokensFavoritesContextType;
+
+export type MarketState = {
```

Comment:
MarketOptionState

Reply:
this comment is a [duplicate of this](https://github.com/gmx-io/gmx-interface/pull/1197#discussion_r1740908724)